Package.resolved

```diff
@@ -0,0 +1,15 @@
+{
+  "originHash" : "1c8e7b22f143659dfd3d9b4d8325e3eacccb1e50f24657cf6a87483bc3d6717a",
+  "pins" : [
+    {
+      "identity" : "purchases-ios-spm",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/RevenueCat/purchases-ios-spm",
+      "state" : {
+        "revision" : "dabbf506344b6f924b44bdfabbaec96bc7943d5d",
+        "version" : "5.3.3"
```

Comment:
Hmm can we automate updating this as well on each release?

Reply:
Good one! This seems to do the trick: https://github.com/RevenueCat/purchases-hybrid-common/pull/922/commits/dfd4d0103d6e2f1b075f9635384e5a78f8affd78

Reply:
we can also not version control package.resolved

Reply:
I don't have a strong opinion, so I Googled some "official" advice. From [swiftlang/swift-evolution](https://github.com/swiftlang/swift-evolution/blob/main/proposals/0175-package-manager-revised-dependency-resolution.md#proposed-solution): 

> Some users may chose to add the `Package.resolved` file to their package's `.gitignore` file. When this file is checked in, it allows a team to coordinate on what versions of the dependencies they should use. If this file is gitignored, each user will separately choose when to get new versions based on when they run the `swift package update` command, and new users will start with the latest eligible version of each dependency. Either way, for a package which is a dependency of other packages (e.g. a library package), that package's `Package.resolved` file will not have any effect on its client packages.

Seems slightly safer to check it in? (Very slightly, because our dependencies have `exact` versions currently.) And it won't affect library consumers apparently.

Reply:
Will merge it for now, but we can always remove it later.